jumpscale/sals/zos/container.py

```diff
@@ -132,3 +134,37 @@ def add_logs(
         container.logs.append(cont_logs)
 
         return cont_logs
+
+    def add_stats(
+        self, container: Container, redis_endpoint: str
+    ) -> ContainerStats:
+        """Enable statistics forwarding for the container
+
+        Args:
+          cont(tfgrid.workloads.reservation.container.1): container instance
```

Comment:
Better to have the type name instead of `tfgrid.workloads.reservation.container.1`?

Reply:
I used example found on the same file. Which type do you expect ?

Reply:
`Container` is expected here, if you're using vscode, `autodocstring` plugin would fill the types from type hints automatically.

Also, the same for the return type, `ContainerStats` is expected.